liota/core/package_manager.py

```diff
@@ -373,10 +379,18 @@ def run(self):
 
             # Switch on message content (command), determine what to do
             command = msg[0]
+            if command in ["unload", "delete", "list", "stat", "unload_all",\
+                           "unload_all", "terminate"]:
+                # currently, write 'Success' to the named pipe once
+                #     successfully receives these commands
+                with open(self._rsp_pipe_file, "w+") as fp:
+                    fp.write('Success\n')
```

Comment:
I think we want this to be declared somewhere global since the client and server need to know about the same strings in the response, just like in the requests.

Reply:
wrap "writing command status to response named pipe" into _write_cmd_exec_status(), where explained responses meaning. The same content is also added in packages/liotad/README.md file.